custom_components/localtuya/cover.py

```diff
@@ -71,7 +88,8 @@ def __init__(
         """Initialize a new LocaltuyaCover."""
         super().__init__(device, config_entry, switchid, **kwargs)
         self._state = None
-        self._position = 50
+        self._current_cover_position = 50
```

Comment:
We have a few of these constants in various platforms with hardcoded values. I believe we should just set them to `None` since we don't know the value yet and wait for an update. It's generally ok to return `None` the the properties to signal that a value is not available.